templates/cron_check_backup.sh.erb

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
This doesn't look like it needs to be an ERB template. Can it move to `files/`?

Reply:
It could. But I think it should stay as a template because:

1. there are no other static script files
2. there is a high chance it will eventually need to be a template.

Reply:
sounds fine